feature/staticroute/static_route_support/README.md

```diff
@@ -0,0 +1,114 @@
+
+## Procedure
+
+#### Initial Setup:
+
+*   Connect DUT port-1, port-2 and port-3 to ATE port-1, port-2 and port-3 respectively
+*   Configure IPv4/IPv6 addresses on DUT and ATE the interfaces
+*   Configure one IPv4 destination i.e. ipv4-network connected to ATE
+*   Configure one IPv6 destination i.e. ipv6-network connected to ATE
+
+### RT-1.26.1 [TODO: https://github.com/openconfig/featureprofiles/issues/2489]
+#### Test to validate static route ECMP
+
+*   Configure IPv4 static routes:
+    *   Configure one IPv4 static route i.e. ipv4-route-a on the DUT for destination ipv4-network with the next hop of port-1
+    *   Configure another IPv4 static route i.e. ipv4-route-b on the DUT for destination ipv4-network with the next hop of port-2
+*   Configure IPv6 static routes:
+    *   Configure one IPv6 static route i.e. ipv6-route-a on the DUT for destination ipv6-network with the next hop of port-1
+    *   Configure another IPv6 static route i.e. ipv6-route-b on the DUT for destination ipv6-network with the next hop of port-2
+*   Initiate traffic from ATE port-3 towards destination ipv4-network and ipv6-network
+*   Validate that traffic is received from DUT on both port-1 and port-2 and ECMP works
+
+### RT-1.26.2 [TODO: https://github.com/openconfig/featureprofiles/issues/2489]
+#### Test to validate static route metric
+
+*   Configure metric of ipv4-route-b and ipv6-route-b to 1000
+    *   /network-instances/network-instance/protocols/protocol/static-routes/static/next-hops/next-hop/config/metric
+*   Validate that the metric is set correctly
+    *   /network-instances/network-instance/protocols/protocol/static-routes/static/next-hops/next-hop/state/metric
+*   Initiate traffic from ATE port-3 towards destination ipv4-network and ipv6-network
+*   Validate that traffic is received from DUT on port-1 only
+
+### RT-1.26.3 [TODO: https://github.com/openconfig/featureprofiles/issues/2489]
+#### Test to validate static route prefrence 
+
+*   Configure preference of ipv4-route-a and ipv6-route-a to 1000
+    *   /network-instances/network-instance/protocols/protocol/static-routes/static/next-hops/next-hop/config/preference
+*   Validate that the preference is set correctly
+    *   /network-instances/network-instance/protocols/protocol/static-routes/static/next-hops/next-hop/state/preference
+*   Initiate traffic from ATE port-3 towards destination ipv4-network and ipv6-network
+*   Validate that traffic is now received from DUT on port-2 only
+
+### RT-1.26.4 [TODO: https://github.com/openconfig/featureprofiles/issues/2489]
+#### Test to validate static route tag
+
+*   Configure a tag of value 10 on ipv4-route-a and ipv6-route-a
+    *   /network-instances/network-instance/protocols/protocol/static-routes/static/config/set-tag
+*   Validate the tag is set
+    *   /network-instances/network-instance/protocols/protocol/static-routes/static/state/set-tag
+
```

Comment:
I think you need to add addtional criteria to make sure the tag can be used:
See [OC description of tag](https://openconfig.net/projects/models/schemadocs/yangdoc/openconfig-network-instance.html#network-instances-network-instance-protocols-protocol-static-routes-static-config-set-tag)

Reply:
This is covered in static to ISIS/BGP redistribution readme that I am working on https://github.com/openconfig/featureprofiles/issues/2494

Reply:
Ok, this is fair.  Tags are only used as part of routing-policy and it is fair to have the tag use case in some other test which include `routing-policy`.